### ISSUE TITLE
Fix Capitalisation On Initial Sync Screen

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-android#4292]
### Action required
- [ ] Update the content in the modal (screenshot) to say: "Initial sync: Importing joined rooms"

### Your use case

On the Initial Sync model there are too many capitals. It's harder to read camel case and does not match other screens in the app
![camel case](https://user-images.githubusercontent.com/89144281/138291172-829b97f1-8ebb-48a9-abb2-c0644fd3a1ee.png)

**Comment by AdamTomaja** [comment]
@daniellekirkwood Hello :) I would like to introduce myself a little bit into the project and fix this issue, but not able to push the feature branch:

```
$ git push --set-upstream origin feature/4292-fix-capitalisation-on-initial-sync-screen
ERROR: Permission to vector-im/element-android.git denied to AdamTomaja.
fatal: Could not read from remote repository.
```

Do I need some permissions? :)
 
Best regards, 
Adam

**Comment by ouchadam** [comment]
:wave: thanks for looking into this!

it looks like you're trying to push directly to `vector-im/element-android` which only vector-im organisation members can currently push to

you'll need to fork the project on github, push the branch and commits to your fork and then raise a pull request from your fork back to `vector-im/element-android`

some docs in case they're helpful https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks

**Comment by AdamTomaja** [comment]
@ouchadam thanks for quick response :)

**Comment by bmarty** [comment]
Thanks @AdamTomaja for your (coming) contribution.

The fix is to replace this string https://github.com/vector-im/element-android/blob/main/vector/src/main/res/values/strings.xml#L205

With
`Initial Sync:\nLoading your conversations\nIf you've joined lots of rooms, this might take a while`

To be iso Element Web.

**Comment by AdamTomaja** [comment]
@bmarty https://github.com/vector-im/element-android/pull/4410 

should i set ```initial_sync_start_importing_account_joined_rooms``` to ```Initial Sync:\nLoading your conversations\nIf you've joined lots of rooms, this might take a whil```" ?

**Comment by bmarty** [comment]
Yes (with a `e` at the end :)), also probably change `Data` to `data` and `Rooms` to `rooms` in other places

**Comment by bmarty** [comment]
Fixed in EA 1.3.7
